lib/teleterm/apiserver/handler/handler_kubes.go

```diff
@@ -24,8 +24,11 @@ import (
 
 	"github.com/gravitational/trace"
 
+	"github.com/gravitational/teleport/api/types"
 	api "github.com/gravitational/teleport/gen/proto/go/teleport/lib/teleterm/v1"
+	"github.com/gravitational/teleport/lib/teleterm/api/uri"
 	"github.com/gravitational/teleport/lib/teleterm/clusters"
+	"github.com/gravitational/teleport/lib/web/ui"
```

Comment:
I'd say let's avoid bringing lib/web into lib/teleterm. Grepping through lib/teleterm, it's not something that we do today.

If we need to reuse some functions between `lib/web` and `lib/teleterm`, I think a better option would to move them to a third package that can be used by both web and teleterm.

What do you think about creating `lib/ui/labels.go` and moving `MakeUILabelsWithoutInternalPrefixes` there? Then we could even drop `UI` from its name.